licenses/bin/jakarta.activation-api.EDL1

```diff
@@ -0,0 +1,13 @@
+Eclipse Distribution License - v 1.0
```

Comment:
Likewise these might not be needed depending on the answer to whether or not this stuff is only used by CI